afk-daily.sh

```diff
@@ -1955,6 +1972,7 @@ run() {
     sleep 1
     checkWhereToEnd
 
+    eval "$currentPos=false" # Prevent loop on error
```

Comment:
Way better solution than the previous one XD

Reply:
I hope!